### PR TITLE
[MeshMoving] add customizable poisson ratio for mesh motion analysis

### PR DESCRIPTION
**Pull request by armingeiser**
The poisson ratio has a quite large effect on the mesh motion.

Larger poisson ratios allow for more shear deformation and therefor a propagation of the mesh motion further into the domain. With this PR the poisson ratio can be modified in the json setting. The default remains 0.3

### CODE REVIEW

**Comment by philbucher** [comment]
@armingeiser there is a test failing in the MeshMovingApp!
Probably the one failing in Fluid is related to that too

File: applications/MeshMovingApplication/custom_elements/structural_meshmoving_element.cpp [hunk 1]
```diff
@@ -117,7 +118,7 @@ StructuralMeshMovingElement::SetAndModifyConstitutiveLaw(
                          // elements; 0 = no stiffening
   const double quotient = factor / detJ0[PointNumber];
   const double weighting_factor = detJ0[PointNumber] * std::pow(quotient, xi);
-  const double poisson_coefficient = 0.3;
+  const double poisson_coefficient = this->pGetProperties()->GetValue(POISSON_RATIO);
```
Comment (philbucher) [inline applications/MeshMovingApplication/custom_elements/structural_meshmoving_element.cpp]:
```suggestion
  const double poisson_coefficient = this->pGetProperties()->Has(POISSON_RATIO) ? this->pGetProperties()->GetValue(POISSON_RATIO) : 0.3;
```

**Review by philbucher — CHANGES_REQUESTED** [review]
without discussing in detail there is sth minor to be fixed otherwise it can fail

we should discuss in person how to implement it

File: applications/MeshMovingApplication/custom_elements/structural_meshmoving_element.cpp (hunk 1 above)
Comment (armingeiser) [reply applications/MeshMovingApplication/custom_elements/structural_meshmoving_element.cpp]: I don't like this because it is a second place where the default value is given.

**Comment by armingeiser** [comment]
Will be discussed in detail in person when @AndreasWinterstein and @philbucher are both here.

File: applications/MeshMovingApplication/custom_elements/structural_meshmoving_element.cpp [hunk 2]
```diff
@@ -18,6 +18,7 @@
 // Project includes
 #include "includes/checks.h"
 #include "includes/mesh_moving_variables.h"
+#include "includes/variables.h"
```
Comment (philbucher) [inline applications/MeshMovingApplication/custom_elements/structural_meshmoving_element.cpp]:
```suggestion
#include "includes/mesh_moving_variables.h"
```

File: applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.cpp [hunk 3]
```diff
@@ -78,11 +78,15 @@ ModelPart* GenerateMeshPart(ModelPart &rModelPart,
...
+
+  // create a new property for the mesh motion
+  Properties::Pointer p_mesh_motion_property = pmesh_model_part->CreateNewProperties(0);
+
   for (int i = 0; i < (int)rModelPart.Elements().size(); i++) {
     ModelPart::ElementsContainerType::iterator it =
         rModelPart.ElementsBegin() + i;
     Element::Pointer p_element = r_reference_element.Create(
-        it->Id(), it->pGetGeometry(), it->pGetProperties());
+        it->Id(), it->pGetGeometry(), p_mesh_motion_property);
```
Comment (philbucher) [inline applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.cpp:89]: didn't we discuss to revert the changes here?

Comment (armingeiser) [reply applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.cpp:89]: :+1: did it only for the le utilities...

Comment (armingeiser) [reply applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.cpp:89]: Sorry wrong, this is the function that creates the mesh model part for the standard mesh solver, here the property needs to be set.

Comment (armingeiser) [reply applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.cpp:89]: Now as we have the `MESH_POISSON_RATIO` variable, we could keep the property of the element as is and just add the `MESH_POISSON_RATIO` at python level in the solver. Then i would not need to change this function and also the arguments of the strategy would not need to be modified.

**Review by philbucher — APPROVED** [review]
